packages/dashboard/src/components/internalDashboard/collapseIcons.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export const PropertiesPanelIcon = () => (
+  <svg
```

Comment:
you should be able to move this into a new file, i have updated dashboard to handle svg files

Reply:
done